README.md

```diff
@@ -128,6 +128,27 @@ var result = await stats.Time("someStat", async t => await GetSomethingAsync());
 
 In all these cases the function or delegate is supplied with a `IDisposableTimer t` so that the stat name can be changed if need be.
 
+#### Configuring in AWS lambda
+
+We are using this library in AWS lambda functions written in C# .Net core. 
```

Comment:
My not-so-inner-pedant thinks this should be `...written in C# using .NET Core`.